stories/SQFormScrollableCard.stories.js

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import * as Yup from 'yup';
+
+import {Paper} from '@material-ui/core';
+import {SQFormScrollableCard, SQFormTextField} from '../src';
+import {createDocsPage} from './utils/createDocsPage';
+
+export default {
+  title: 'Forms/SQFormScrollableCard',
+  component: SQFormScrollableCard,
+  argTypes: {
+    onSave: {action: 'onSave', table: {disable: true}},
+    children: {table: {disable: true}},
+    validationSchema: {table: {disable: true}}
+  },
+  parameters: {
+    docs: {page: createDocsPage({showStories: false})}
+  }
+};
+
+const defaultArgs = {
+  title: 'Default',
+  initialValues: {hello: ''},
+  muiGridProps: {
+    spacing: 2,
+    alignItems: 'center'
+  },
+  onSubmit: () => {},
+  validationSchema: {hello: Yup.string().required('Required')}
+};
+
+const Template = (args = {}) => {
```

Comment:
This default object assignment needs to be removed to allow the Storybook Controls to work